### DISCUSSION TITLE
Requesting a code tweak for forecasted low alert

### DISCUSSION THREAD

**Discussion by gth001** [NightscoutFoundation/xDrip#2248] [Ideas]
It would be helpful to me if I could get more than 1 predicted low alert.
I get an alert 60 minutes before a low, but I would like more options.
How about "re-raise alarm every ____ minutes if low still predicted"?
Alternatively, make the mins variable a list, where I can put "60,45,30,15,10,5" for example.
It would be nice, since I sometimes forget to keep an eye on my dropping blood sugar.
![Screenshot_20220729-162915_xDrip_](https://user-images.githubusercontent.com/10224533/181860586-67b190fc-a6a4-4133-8ae0-cc83e1c42c9d.jpg)

**Comment by Navid200** [comment]
The Forecast Low alert is only an extrapolation.  It's not a simulator.  It's just a numerical analysis performed solely on the readings.  It completely ignores your carbs or insulin.

Setting it up to predict 60 minutes is really not a very good idea.

The forecast low alert is meant to complement your low level alerts.  It's not meant to be the only type of alert you rely on.

It is possible I am missing a very important need you have felt.  Please explain.

We have some important issues with alerts (in my opinion of course) and I would like those to be addressed.  For example, the forecast low alert will not be heard at all if you accidentally have lowered your ring and notification volume.  In that case, it will make no difference how many of them you have. 
I would like that to be changed so that the alert can use the volume profile.  But, if we open new feature requests for alerts, it will only distract the developers.

Still, please explain why this is so important.
Thanks

**Comment by gth001** [comment]
> It completely ignores your carbs or insulin.

Oh, I didn't know that. That is OK though, since I usually don't input carbs and insulin. 

I do find the 60 minute predicted low alert very helpful though :). The omnipod 5 algorithm also predicts 60 minutes (not adjustable) ahead when it suspends insulin, but if my IOB is too high, I will need to be eating too. When I get that notification - I check my IOB on the omnipod 5 apps notification. If it is enough insulin to correct me beyond low, I can snack on healthy things for the next hour to prevent the low. They work together very well.

The only problem is the notification disappears when I click it - It'd just be nice to persist the entire time the low is predicted.

I would call this a lower priority suggestion. Perhaps if they are re-writing the code already, this suggestion will come to mind. If the developers have important things pending, it is fine to set this aside. 

>The forecast low alert is meant to complement your low level alerts. It's not meant to be the only type of alert you rely on.

You're right, I should set some more low alarms at certain levels, I'll do that.

**Comment by jonprogers** [comment]
The 'forecast low' tends to be my most useful alert as it's the one that tells me something is going wrong when there is still time to do something about it rather than something has gone wrong already!! I can, therefore, see why the OP started this discussion as I think this would be useful too.

For me this would be a lower priority than fixing existing known issues but to be able to set multiple forecast low alerts at different 'time to low' periods in the same way as I can currently set multiple low and high alerts at different BG levels in the 'Alarms and Alerts/Glucose Level Alerts List' menu would be a feature I would definitely use.

This may be a step too far but to be able to set both the time and level would really play to my control freakiness well - so setting a list of alerts something like 'Raise an alert when my BG is predicted to fall below 4mmol/L in the next 20 minutes' or 'Raise an alert when my BG is predicted to rise above 12mmol/L in the next 40 minutes' would give me time to assess if and how I needed to react when predicted to leave range in either direction.

As a thought for the OP - in trying to make best use of the existing functionality I have tried a number of different settings for 'Alarm at Forecasted Low mins' and agree that 60 mins was very easy to ignore, thinking I'll check again in a few mins and then never actually doing so. My sweet spot is around 35 mins to low, for me this is early enough to stand a chance of avoiding the hypo but late enough that I can normally see what I need to do, there and then, so it avoids me procrastinating and then doing nothing before I get the low alert! There is so much flexibility already provided by the existing settings its great to be able configure the tech to support my particular personality flaws!!!

**Comment by Navid200** [comment]
@tolot27 @tzachi-dar 
Should this (multiple forecast low alerts) become a feature request?
